core/tests/test_utils_test.py

```diff
@@ -428,6 +429,15 @@ def mock_getcwd():
 class EmailMockTests(test_utils.EmailTestBase):
     """Class for testing EmailTestBase."""
 
+    def test_override_run_swaps_contexts(self):
+        """Test that the current_function
+        email_services.send_email_to_recipients() is correctly swapped to its
+        mock version when the testbase extends EmailTestBase.
+        """
+        currrent_function = getattr(email_services, 'send_email_to_recipients')
```

Comment:
"current" is misspelled

Perhaps called_function or referenced_function is a better name though.

Reply:
Done